### PR TITLE
rtmp-services: Add Streamway to ingest list

### PR DESCRIPTION
**Pull request by Ramjivan**
### Description
Adding Streamway to OBS-Studio's Ingest List

### Motivation and Context
Streamway (https://streamway.in) is a re-streaming and CDN provider, that allows users to livestream on multiple social media platforms like YouTube, Facebook, LinkedIn, Twitch, Twitter and many more at same time, so broadcasters can get more views and maximise their autocancel.  Streamway also acts as a live CDN to allow Broadcasters to publish livestream directly on their custom Mobile app or websites. Streamway is wildly used in India with more than 3 years of availability.

By adding this website to the ingest list, OBS-Studio users will be able to easily and conveniently use Streamway as a source for their live streams and content.

### How Has This Been Tested?
- [x] JSON validation: Ensured that the JSON files are well-formed and adhered to the required format for OBS-Studio's ingest list.
- [x] Copy JSON files to %APPDATA%\obs-studio\plugin_config\rtmp-services: Manually copied the JSON files to the appropriate location where OBS-Studio reads ingest settings.
- [x] Checked settings: Verified that "Streamway" appears in the updated ingest list within OBS-Studio's settings.
- [x] Connection testing: Attempted to connect to the servers provided by "Streamway" to ensure successful streaming functionality.
- [x] According to [Service Submission Guidelines](https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines)

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/package.json
Comment (gxalpha) [inline plugins/rtmp-services/data/package.json:8]: Please also update the version 4 lines above.

Comment (Ramjivan) [reply plugins/rtmp-services/data/package.json:8]: Updated version

**Comment by Ramjivan** [comment]
Hello @Fenrirthviti @Lain-B  @RytoEX ,

I hope you're all well. Could you please take a moment to look at this PR when you have the time?
I can confirm that I am the representative of Streamway. 

Thank you!

**Comment by Fenrirthviti** [comment]
We will review when we have time, thanks!

**Review by Fenrirthviti — APPROVED** [review]

**Comment by Ramjivan** [comment]
Hi @RytoEX , just wanted to follow up on this pull request. It's been approved by @Fenrirthviti. Are there any other outstanding items before this can be merged? Thanks!

**Comment by RytoEX** [comment]
This PR has merge conflicts, so it cannot be merged as-is. Unfortunately, this PR also has maintainer edits disabled, so I cannot fix them myself.

Additionally, do not use merge commits to keep your branch up to date. Use `git rebase` instead.

**Comment by Ramjivan** [comment]
Hi @RytoEX 

Do I need to increment version in `package.json` again ? 

You do not have write access because of [GitHub issue ](https://github.com/orgs/community/discussions/5634) related to org. account.
However I have added you as a collaborator, so you have direct access now.

**Comment by RytoEX** [comment]
> Do I need to increment version in `package.json` again ?
> 
> You do not have write access because of [GitHub issue ](https://github.com/orgs/community/discussions/5634) related to org. account. However I have added you as a collaborator, so you have direct access now.

Yes, you would need to increment it again and remove the merge commits.

**Comment by RytoEX** [comment]
> You do not have write access because of [GitHub issue ](https://github.com/orgs/community/discussions/5634) related to org. account. However I have added you as a collaborator, so you have direct access now.

Thanks. You can remove me from the repo now.

**Comment by Ramjivan** [comment]
Thank You for merge, You guys have been extremely helpful in the process😊

**Pull request merged**
